caffe2/ideep/operators/conv_op.cc

```diff
@@ -36,21 +34,6 @@ class IDEEPConvOp final : public IDEEPConvPoolOpBase {
         "*",
         group_);
 
-    bool weights_changed =
```

Comment:
Same here

Reply:
The filter has been converted to optimized format already in optimizeForIdeep.
Nothing needs to do here for inference.